app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -295,7 +280,7 @@ public void onConnectionSuspended(int i) {
     }
 
     @Override
-    public void onConnectionFailed(ConnectionResult result) {
+    public void onConnectionFailed(@NonNull ConnectionResult result) {
```

Comment:
See https://github.com/gdg-x/frisbee/pull/560/files#r55929152 for the same question.